ObjectiveGit/GTRepository+Pull.m

```diff
@@ -20,6 +20,8 @@
 #import "GTIndexEntry.h"
 #import "git2/errors.h"
 
+NSString * const GTPullMergeConflictedFiles = @"GTPullMergeConflictedFiles";
```

Comment:
We should expose this from the header too, so users don't have to know the string literal.